frontend/taipy-gui/packaging/taipy-gui.d.ts

```diff
@@ -117,6 +117,21 @@ export interface TableProps extends TaipyPaginatedTableProps {
 }
 export declare const Table: (props: TableProps) => JSX.Element;
 
+export interface FilterDesc {
+    col: string;
+    action: string;
+    value: string | number | boolean | Date;
```

Comment:
Will this be a problem in the future when supporting filters on enumerated values?
Or string will do uh?

Reply:
I guess it will do as value holds the choice of the user
The enumeration will have to go into the colDesc